packages/docs-page/index.tsx

```diff
@@ -19,7 +22,20 @@ import LoadingSkeleton from './components/loading-skeleton'
 import useIsMobile from './use-is-mobile'
 import s from './style.module.css'
 
-export function DocsPageWrapper({
+interface DocsPageWrapperProps {
+  canonicalUrl: string
+  description: string
+  navData: NavData
+  currentPath: string
+  pageTitle: string
+  baseRoute: string
+  githubFileUrl: string
+  product: { name: string; slug: string }
+  showEditPage: boolean
+  versions: { name: string; label: string }[]
+}
+
+const DocsPageWrapper: ComponentType<DocsPageWrapperProps> = ({
```

Comment:
We can refine this type further and use `FunctionComponent`. The `ComponentType` type also includes `ComponentClass`, which isn't the case here!

ref: https://sourcegraph.com/github.com/DefinitelyTyped/DefinitelyTyped/-/blob/types/react/index.d.ts?L82:10

Reply:
TIL!